### PR TITLE
disable_dependent_services argument/variable have been added

### PR DESCRIPTION
**Pull request by alexkonkin**
The argument/variable that manages disable_dependent_services has been added

### CODE REVIEW

File: modules/project_services/variables.tf [hunk 1]
```diff
@@ -33,3 +33,10 @@ variable "disable_services_on_destroy" {
   default     = "true"
   type        = "string"
 }
+
+variable "disable_dependent_services" {
```
Comment (kopachevsky) [inline modules/project_services/variables.tf]: I guess we need to regenerate documentation in this case?

File: modules/project_services/main.tf [hunk 2]
```diff
@@ -22,4 +22,5 @@ resource "google_project_service" "project_services" {
   project            = "${var.project_id}"
   service            = "${element(var.activate_apis, count.index)}"
   disable_on_destroy = "${var.disable_services_on_destroy}"
+  disable_dependent_services = ${var.disable_dependant_services}
```
Comment (kopachevsky) [inline modules/project_services/main.tf]: no double quotes around

File: modules/project_services/variables.tf [hunk 3]
```diff
@@ -33,3 +33,10 @@ variable "disable_services_on_destroy" {
...
+  description = "Whether services that are enabled and which depend on this service should also be disabled when this service is destroyed. https://www.terraform.io/docs/providers/google/r/google_project_service.html#disable_dependent_services"
```
Comment (kopachevsky) [inline modules/project_services/variables.tf]: very unclear description

Comment (alexkonkin) [reply modules/project_services/variables.tf]: This description comes directly from the terraform documentation related to this 
argument,
https://www.terraform.io/docs/providers/google/r/google_project_service.html#disable_dependent_services

What exactly is not clear for you?
Thanks.

**Comment by aaron-lane** [comment]
Fixes #168

**Comment by aaron-lane** [comment]
This branch must be rebased against master once #189 is merged.

```bash
git remote add upstream https://github.com/terraform-google-modules/terraform-google-project-factory.git
git fetch upstream
git checkout master
git merge upstream/master
git checkout bug/project_add_disable_dependent_services
git rebase master
git push --force
```

**Pull request merged**